testing/integration/upgrade_test.go

```diff
@@ -111,9 +118,16 @@ func TestFleetManagedUpgrade(t *testing.T) {
 	fleetServerURL, err := tools.GetDefaultFleetServerURL(kibClient)
 	require.NoError(t, err)
 
+	parsedFromVersion, err := version.ParseVersion(fromVersion)
+	require.NoError(t, err)
+
 	t.Log("Enrolling Elastic Agent...")
+	var nonInteractiveFlag bool
+	if version_8_2_0.Less(*parsedFromVersion) {
```

Comment:
Nit: could just be a personal issue, but I find the opposite check to be more readable. Of course, that would mean implementing `GreaterOrEqual` as a convenience method 🙂.
 
```suggestion
		if parsedFromVersion.GreaterOrEqual(*version_8_2_0) {
```

Reply:
We can do that in a follow-up PR ;)